MediaRecorder.html

```diff
@@ -326,12 +313,6 @@
             playable, but the combination of all the Blobs from a completed
             recording MUST be playable.</p>
 
-            <p> If any Track within the MediaStream is muted at any time (i.e.,
```

Comment:
I think you should keep the "record black and silence" text, just to be explicit.